content/en/Platform Deep Dive/Scans/target_auth.md

```diff
@@ -103,3 +104,18 @@ To wait for a login input/element when the target has some unusual behavior whil
 - The login page exhibits unusual behavior, requiring the use of the "hack" `1_wait` to wait for a specific input (this issue can be particularly difficult to identify).
 - The target is blocking our access, preventing us from reaching the login page.
 - The login process includes a required CAPTCHA that users may not notice because the page uses "smart recaptcha," which is only triggered when a crawler is detected.
+
+# Using a Login Sequence
+
+If your login page does not have all the login credentials inputs in one page, for example, you have to enter an email then click next to enter the password, you can use a login sequence. It will record your actions and replay them during the scan. 
+
+To learn more about Cobalt's Sequence Recorder browser plugin, check the [Sequence Recorder](/platform-deep-dive/scans/sequence-recorder) page.
+
+Once you have a sequence recorded, go to the target's Advanced Settings, toggle on the “Login sequence option and then:
+1. Enter a name for the sequence to easily identify it later.
+2. Paste the sequence in the field provided, or upload the sequence file directly.
+3. Click on the "Add Sequence" button.
```

Comment:
This part is already mentioned in the other page, I would leave it only 1 of both places to avoid repetition

Reply:
I saw the last line in the sequence recorder page, but it was a brief line mentioning where the generated file is used. 
I thought it would be nice to have the clear steps here as the main guide.